oauth-server/test/config/httpgw.yaml

```diff
@@ -0,0 +1,69 @@
+Address: 0.0.0.0:7000
```

Comment:
why is httpgw config part of the oauth server? and /home/krz/go is probably not what you wanted to commit :)

Reply:
removed